src/file.ts

```diff
@@ -2180,6 +2214,8 @@ class File extends ServiceObject<File> {
    * @param {object} config Configuration object.
    * @param {string} config.action "read" (HTTP: GET), "write" (HTTP: PUT), or
    *     "delete" (HTTP: DELETE), "resumable" (HTTP: POST).
+   * @param {string} [config.version='v2'] The signing version to use, either
```

Comment:
Add a note that when using resumable (POST). The signature includes `x-goog-resumable:start`.